ragelo/evaluators/answer_evaluators/pairwise_evaluator.py

```diff
@@ -148,8 +149,8 @@ def _process_answer(self, llm_response: LLMResponseType) -> LLMResponseType:
         """Extracts the relevant part of an answer."""
         if isinstance(llm_response.parsed_answer, dict):
             answer = llm_response.parsed_answer["winner"]
-        elif isinstance(llm_response.parsed_answer, PairWiseAnswerAnswerFormat):
-            answer = llm_response.parsed_answer.winner
+        elif isinstance(llm_response.parsed_answer, PydanticBaseModel):
```

Comment:
is this because of deserialization?

Reply:
Not sure how it got here. Moving it back to how it was.